### PR TITLE
feat(parser): support JSX syntax.

### PR DESCRIPTION
**Pull request by ama-ch**

### CODE REVIEW

**Comment by ama-ch** [comment]
@teppeis Here's support of JSX syntax. Could you review this?

File: lib/parser.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 var parse = require('espree').parse;
 var doctrine = require('doctrine');
 var _ = require('underscore');
-var traverse = require('estraverse').traverse;
+var traverse = require('estraverse-fb').traverse;
 var Syntax = require('estraverse').Syntax;
```
Comment (teppeis) [inline lib/parser.js]: Use `require('estraverse-fb').Syntax` to ensure it's JSX-injected Syntax.
Fix `lib/visitor.js` too.

File: lib/parser.js [hunk 2]
```diff
@@ -66,7 +66,10 @@ Parser.prototype.parse = function(src) {
     comment: true,
     attachComment: true,
     loc: true,
-    ecmaVersion: 6
+    ecmaVersion: 6,
+    ecmaFeatures: {
+      jsx: true
```
Comment (teppeis) [inline lib/parser.js]: Enabling JSX by default is a bit aggressive, but it's ok! :smiling_imp: 
We need `.fixclosurerc.json` feature to load such a nested options easier.

File: lib/parser.js (hunk 1 above)
Comment (teppeis) [inline lib/parser.js]: Fixed 6485414

**Comment by teppeis** [comment]
LGTM!

**Comment by teppeis** [comment]
I'll publish a new version.

**Comment by teppeis** [comment]
v1.5.0 is released! :rocket:

**Comment by ama-ch** [comment]
Thank you for fixing :yum:

**Pull request merged**